### ISSUE TITLE
Digital key value

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BNETDocs/bncsutil#4]
```
http://code.google.com/p/bncsutil/source/browse/trunk/src/bncsutil/cdkeydecoder.
cpp#480

Should be:
value1 = LSB4(*(uint32_t*) (((char*) values) + 2)) & 0xFFFFFF03;

According to:
http://forum.valhallalegends.com/index.php/topic,18165.0.html
```

Original issue reported on code.google.com by `AHeinerm` on 6 Sep 2010 at 3:42

**Comment by carlbennett** [comment]
Permalink to source code:

https://github.com/BNETDocs/bncsutil/blob/6cbe0bd95d625bd1c3d4aa2ab8996da81d3c8dd9/src/bncsutil/cdkeydecoder.cpp#L480

**Comment by carlbennett** [comment]
Bug reported by @heinermann.

**Comment by carlbennett** [comment]
Resolved in 3c784bfd8ab690941c51ccb91964c6d4132834c6.
